neuralmonkey/decoders/ctc_decoder.py

```diff
@@ -90,7 +92,7 @@ def cost(self) -> tf.Tensor:
     def logits(self) -> tf.Tensor:
         vocabulary_size = len(self.vocabulary)
 
-        encoder_states = self.encoder.hidden_states
+        encoder_states = self.encoder.temporal_states
 
         weights = tf.get_variable(
             name="state_to_word_W",
```

Comment:
řádka o 1 pod timhle komentem se mi nezdá. `encoder_states.shape[2]` ?? nemělo by to bejt `tf.shape` nebo `get_shape`?

Reply:
`tf.shape` je blbost, to vrací tensor a ta matice musí vzniknout dřív, než je možný nějakej tensor vůbec evaluovat. Jinak k mému překvapení platí `encoder_states.get_shape() == encoder_states.shape`, asi se to někdy přidalo TF, aniž bych to vědět, vrací to `TensorShape`.

Reply:
No jo, v dokumentaci 1.4 píšou, že `Tensor.get_shape()` je alias pro `Tensor.shape`. Nechcem to tady upravit, abychom byli konzistentní? Jinak je mi to fuk.

Reply:
Já bych na to spíš kašlal.